.github/workflows/stale_issue.yml

```diff
@@ -0,0 +1,45 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
I'm a bit curious what `cron: "0 0 * * *"` means here

Reply:
It is a [crontab](https://man.archlinux.org/man/crontab.5.en) expression. In this case it is expressing that at the 0th minute, of the 0th hour (12 AM), run this expression every day, for every month, for every day of the week.